beta-src/src/enums/ArrowColor.ts

```diff
@@ -0,0 +1,11 @@
+enum ArrowColor {
```

Comment:
Can we just have this Enum in the ArrrowType.ts file or would we prefer to have a single Enum per file?

Reply:
Generally we prefer a single enum per file so that we don't have to be hunting for enums in multiple files where they might be.